vdev/src/commands/install_git_hooks.rs

```diff
@@ -0,0 +1,40 @@
+use anyhow::{Result, Ok};
```

Comment:
I think this doesn't need to be at the top level of `vdev`, maybe better suited to be under `meta`?

Reply:
moved to meta